backend/tests/unit/git/conftest.py

```diff
@@ -660,9 +659,7 @@ async def gql_query_data_02() -> dict:
 
 @pytest.fixture
 async def mock_schema_query_01(helper, httpx_mock: HTTPXMock) -> HTTPXMock:
-    response_text = Path(os.path.join(helper.get_fixtures_dir(), "schemas", "schema_01.json")).read_text(
-        encoding="UTF-8"
-    )
+    response_text = Path(helper.get_fixtures_dir(), "schemas", "schema_01.json").read_text(encoding="UTF-8")
```

Comment:
Again `helper.get_fixtures_dir()` is a `Path` object, weird that it was used with `os.path.join`